src/main/python/main/ayab/engine/control.py

```
@@ -232,6 +234,10 @@ def cnf_line_API6(self, line_number: int) -> bool:
             + " pat_row: "
             + str(self.pat_row)
         )
+        try:
+            msg = msg + " memo: " + str(self.memos[self.pat_row])
+        except:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)